apps/web/src/pages/AnnouncementsPage/AnnouncementsPage.tsx

```diff
@@ -0,0 +1,129 @@
+import * as React from "react";
+import { MessageDescriptor, defineMessage, useIntl } from "react-intl";
+import MegaphoneOutlineIcon from "@heroicons/react/24/outline/MegaphoneIcon";
+import MegaphoneSolidIcon from "@heroicons/react/24/solid/MegaphoneIcon";
+import { useQuery, useMutation } from "urql";
+
+import { SitewideAnnouncementInput, graphql } from "@gc-digital-talent/graphql";
+import { Pending, IconType } from "@gc-digital-talent/ui";
+import { toast } from "@gc-digital-talent/toast";
+import { commonMessages } from "@gc-digital-talent/i18n";
+
+import SEO from "~/components/SEO/SEO";
+import useRoutes from "~/hooks/useRoutes";
+import AdminContentWrapper from "~/components/AdminContentWrapper/AdminContentWrapper";
+import AdminHero from "~/components/Hero/AdminHero";
+
+import SitewideAnnouncementSection from "./SitewideAnnouncementSection";
+
+const AnnouncementPage_Query = graphql(/* GraphQL */ `
+  query AnnouncementPage {
+    sitewideAnnouncement {
+      isEnabled
+      publishDate
+      expiryDate
+      title {
+        en
+        fr
+      }
+      message {
+        en
+        fr
+      }
+    }
+  }
+`);
+
+const UpdateSitewideAnnouncement_Mutation = graphql(/* GraphQL */ `
+  mutation UpdateSitewideAnnouncement(
+    $sitewideAnnouncementInput: SitewideAnnouncementInput!
+  ) {
+    updateSitewideAnnouncement(
+      sitewideAnnouncementInput: $sitewideAnnouncementInput
+    ) {
+      isEnabled
+      publishDate
+      expiryDate
+      title {
+        en
+        fr
+      }
+      message {
+        en
+        fr
+      }
+    }
+  }
+`);
+
+export const pageTitle: MessageDescriptor = defineMessage({
+  defaultMessage: "Announcements",
+  id: "ll9u08",
+  description: "Page title for the announcements page",
+});
+export const pageOutlineIcon: IconType = MegaphoneOutlineIcon;
+export const pageSolidIcon: IconType = MegaphoneSolidIcon;
+
+const AnnouncementsPage = () => {
+  const intl = useIntl();
+  const routes = useRoutes();
+  const formattedPageTitle = intl.formatMessage(pageTitle);
+
+  const [{ data: initialData, fetching: queryFetching, error: queryError }] =
+    useQuery({
+      query: AnnouncementPage_Query,
+    });
```

Comment:
Not a huge deal since this is likely to only happen once but it does not appear without a refresh the _very_ first time. You could just add conext.

```tsx
 useQuery({
   context: { additionalTypenames: ["SitewideAnnouncement"],
 })
```

Reply:
Oh, good catch!  I didn't notice that this was happening.

Reply:
Oh yeah, that's cleaner.
[move out memo](https://github.com/GCTC-NTGC/gc-digital-talent/pull/9387/commits/592ad8da04be8032284592b11a5925354db80464)